tools/cli/help.txt

```diff
@@ -440,26 +440,22 @@ commands can be accessed by pressing the up arrow.
 
 
 >>> mongo
-Connect to the Mongo database for the specified site.
+Connect to the local Mongo database
 Usage: meteor mongo [--url]
 
 Opens a Mongo shell to view or manipulate collections.
 
-If site is specified, this is the hosted Mongo database for the deployed
-Meteor site.
-
-If no site is specified, this is the current project's local development
-database. In this case, the current working directory must be a
-Meteor project directory, and the Meteor application must already be
-running.
-
 Instead of opening a shell, specifying --url (-U) will return a URL
 suitable for an external program to connect to the database. For remote
 databases on deployed applications, the URL is valid for one hour.
 
 Options:
-  --url, -U  return a Mongo database URL
+  --url, -U return a Mongo database URL
 
+Currently, this feature is only for local developments and the Mongo shell
```

Comment:
`local developments` doesn't seem quite right in this context. Maybe re-word this to something like:

> Currently, this feature can only be used when developing locally. The opened Mongo shell connects to the current project's local development database. The current working directory must be a Meteor project directory, and the Meteor application must already be running.